cli/src/main.rs

```diff
@@ -88,13 +90,13 @@ async fn sign_up(pool: &PgPool, username_opt: Option<String>) -> color_eyre::Res
 
 async fn add_user(pool: &PgPool, username: &str) -> color_eyre::Result<()> {
     let user_id = db::create_user(pool, username).await?;
-    persist_auth_session(user_id).await?;
+    persist_auth_session(user_id)?;
 
     println!("User {username} added successfully with ID {user_id}!");
     Ok(())
 }
 
-async fn persist_auth_session(user_id: Uuid) -> color_eyre::Result<()> {
```

Comment:
This was the 'last' clippy that I ran into when I turned off that panic lint

It was saying that this async wasn't needed since there were no `awaits` in the function! So removed it and removed the `.await` from where we call this function